src/app/clients/clients-view/view-survey/take-survey/take-survey.component.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
can you please comment the entire file although not necessary but it's a good practice.